application/helpers/SurveyRuntimeHelper.php

```diff
@@ -1729,6 +1729,15 @@ private function setSurveySettings($surveyid, $args)
         extract($args);
 
         $this->aSurveyInfo                 = getSurveyInfo($this->iSurveyid, App()->getLanguage());
+        if (isset($args['popuppreview']) && $args['popuppreview']) {
+            $this->aSurveyInfo['showxquestions'] = 'N';
+            $this->aSurveyInfo['shownoanswer'] = 'N';
```

Comment:
Why ?

Reply:
We had a task for the cloud version to make sure that we have a readonly preview for surveys for new users shown in a popup and these changes were implemented for that purpose. Here, in this repo we included the support for this feature due to a recent decision according to which we want to simplify code maintenance for the future.

Reply:
Why for shownoanswer to false ?

The preview are not the same than reality in this case. More for a printopdf system ?

Reply:
"The preview are not the same than reality in this case."

Yes, the intention was to have this difference. If you have some ideas for improvement, then please present them to Nathalie or Jory, both of them checked this popup preview and approved it.

Reply:
Again don't have access to MW-728 project part.